docs-src/content/connecting/_index.md

```diff
@@ -285,7 +291,16 @@ for mdl in model_list:
 {{< /tab >}}
 
 {{< tab "C++" "c++" >}}
-// Example coming soon!
+// Request the list of available models
+auto modelList = client.listModels();
+std::cout << "\nAvailable Models:" << std::endl;
```

Comment:
In the context of a full application from which this example code derives, it was helpful to have the extra newline. I didn't use `std::endl` because I didn't really care for the extra flush that comes with it right then (completely arbitrary decision - performance wise it won't really matter either way).